src/Controllers/PushController.js

```diff
@@ -84,6 +88,34 @@ export class PushController {
     }
     return expirationTime.valueOf();
   }
+
+  /**
+   * Get push time from the request body.
+   * @param {Object} request A request object
+   * @returns {Number|undefined} The push time if it exists in the request
+   */
+  static getPushTime(body = {}) {
```

Comment:
could we add a test for that piece?

There is a bit of logic there, even trivial but that would help not introduce any regression. 

I could add them on the top of your branch if you don't have much time :)